js/components/avatar.vue

```diff
@@ -0,0 +1,14 @@
+<template>
+  <a class="avatar" :href="user.url" :title="user | display">
+      <img v-if="user.avatar" :src="user.avatar"
+          class="avatar" :alt="user | display" width="52" height="52">
```

Comment:
I think `width` and `height` should be binded to a common size parameter which default to 52.